src/Microsoft.Windows.CsWin32/templates/BOOL.cs

```diff
@@ -1,6 +1,11 @@
 ﻿partial struct BOOL
 {
+	public const int Size = sizeof(int);
```

Comment:
We only use it in DragDropFormatTests. I'd say its probably because Marshal.SizeOf is evaluated at runtime and so can't be const.

Reply:
If it's just for tests, I'll suggest you use `sizeof(int)` (knowing we'll never switch from the 4 byte size) or just define the const yourself.

Reply:
Here is where its used, in a PInvoke.GlobalAlloc call for test data.
https://github.com/dotnet/winforms/blob/7bfbde5453eeeaea582006a683f247f6912fc7ac/src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DragDropFormatTests.cs#L25-L32

Reply:
That looks like a memory leak to me, since the code calls `GlobalAlloc` but never calls `GlobalFree`. 
I generally avoid using `GlobalAlloc` in favor of simply pointing to a local bool variable, or allocating a managed array and pinning using `fixed` across a native call so I don't have to worry about memory leaks like this.
It doesn't look like this memory is treated as a bool anyway. Later that same field is set to an `IUnknown` pointer.